tests/linalg_test.py

```diff
@@ -1373,6 +1373,84 @@ def expm(x):
         return jsp.linalg.expm(x, upper_triangular=False, max_squarings=16)
       jtu.check_grads(expm, (a,), modes=["fwd", "rev"], order=1, atol=tol,
                       rtol=tol)
+  @parameterized.named_parameters(
+        jtu.cases_from_list({
+            "testcase_name":
+            "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+            "shape": shape, "dtype": dtype
+        } for shape in [(4, 4), (15, 15), (50, 50), (100, 100)]
+                            for dtype in float_types + complex_types))
+  @jtu.skip_on_devices("gpu", "tpu")
```

Comment:
Are these operations unsupported, slow or inaccurate for GPU/TPU? It would be good to understand why all these tests are skipped.

Reply:
The bottleneck is the schur primitive which is only implemented for a CPU backend. What could be done is to test `_sqrtm_triu` on its own, on randomly generated upper triangular matrices (to skip the call to `shur`) so at least one test would be performed on GPU and TPU

Reply:
Ah, that makes complete sense. No need to change anything then. I don't think it's worth testing `_sqrt_triu` separately.